src/Popover.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
   import { onMount } from 'svelte';
-  import { createPopper } from '@popperjs/core/dist/esm/popper';
```

Comment:
Hi, the rest of this PR looks good, however changing this line causes issues in svelte repl and in sapper last I checked.  Is this necessary to change this?